draft-ietf-quic-transport.md

```diff
@@ -2460,26 +2460,31 @@ of the packet header.  The remainder of the first byte and an arbitrary number
 of bytes following it that are set to unpredictable values.  The last 16 bytes
 of the datagram contain a Stateless Reset Token.
 
-To entities other than its intended recipient, a stateless reset will appear
-to be a packet with a short header.  For the stateless reset to appear as a
-valid QUIC packet and be smaller than the received packet, the Unpredictable
-Bits field needs to include at least 46 bits of data (or 6 bytes, less the
-two fixed bits).  To ensure the stateless reset packet is not smaller than
-other packets received on the connection, an endpoint SHOULD also ensure the
-total packet length is at least the minimum chosen CID length plus 22 bytes.
-22 bytes allows for 1 type byte, 4 packet number and data bytes,
-16 bytes for AEAD expansion, and an extra byte to allow the peer to
-send a smaller stateless reset than the packet it receives.
-
-The Stateless Reset Token corresponds to the minimum expansion of the
-packet protection AEAD.  More unpredictable bytes might be necessary if the
-endpoint could have negotiated a packet protection scheme with a larger minimum
-AEAD expansion.
-
-An endpoint SHOULD NOT send a stateless reset that is significantly larger than
-the packet it receives.  Endpoints MUST discard packets that are too small to be
-valid QUIC packets.  With the set of AEAD functions defined in {{QUIC-TLS}},
-packets that are smaller than 21 bytes are never valid.
+To entities other than its intended recipient, a stateless reset will appear to
+be a packet with a short header.  For the stateless reset to appear as a valid
+QUIC packet, the Unpredictable Bits field needs to include at least 38 bits of
+data (or 6 bytes, less the two fixed bits).
+
+A minimum size of 21 bytes does not guarantee that a stateless reset is
+difficult to distinguish from other packets if the recipient requires the use of
+a connection ID.  In order to ensure that a stateless reset that appears to be
```

Comment:
This second sentence is hard to follow.  How about "To prevent a resulting stateless reset from being distinguished from a valid packet, all packets ..."